components/VM_Arm/src/main.c

```diff
@@ -395,6 +395,46 @@ static int camkes_vm_utspace_alloc_at(void *data, const cspacepath_t *dest, seL4
 
 }
 
+static bool add_uts(vka_t *vka, seL4_CPtr cap, uintptr_t paddr, size_t size,
+                    bool is_unrestricted)
```

Comment:
is_device is probably a better flag to use as it more closely aligns with both the `simple_` interface and the `seL4_Bootinfo_t` untyped definitions?

Reply:
Good point, I've changed it to use `is_device`.